src/pages/settings/Payments/PaymentMethodList.js

```diff
@@ -210,12 +212,16 @@ class PaymentMethodList extends Component {
     render() {
         return (
             <>
-                <FlatList
-                    data={this.getFilteredPaymentMethods()}
-                    renderItem={this.renderItem}
-                    keyExtractor={item => item.key}
-                    ListEmptyComponent={this.renderListEmptyComponent()}
-                />
+                {this.props.isLoadingPaymentMethods && !this.props.network.isOffline ? (
```

Comment:
instead of `&& !this.props.network.isOffline`, what about wrapping ActivityIndicator in a FullPageOfflineBlockingView?

Reply:
I think when we discussed offline behavior for the payments page, we decided not to do any sort of blocking pattern.